### PR TITLE
chore(data-list): updated expandable demo to include expand/collapse all in toolbar

### PR DESCRIPTION
**Pull request by mcoker**
fixes https://github.com/patternfly/patternfly/issues/4767

### CODE REVIEW

File: src/patternfly/demos/DataList/examples/DataList.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ wrapperTag: div
       {{> toolbar-template
             toolbar--id="data-list-expandable-example-toolbar"
             toolbar-template--HasToggleGroup="true"
+            toolbar-template--HasExpandAll="true"
```
Comment (mattnolting) [inline src/patternfly/demos/DataList/examples/DataList.md:65]: 👍 👍

File: src/patternfly/demos/Toolbar/toolbar-template.hbs [hunk 2]
```diff
@@ -25,6 +26,13 @@ Options: [
       {{!-- toggle group --}}
       {{#if toolbar-template--HasToggleGroup}}
         {{#> toolbar-group toolbar-group--modifier="pf-m-toggle-group pf-m-show-on-xl"}}
+          {{#if toolbar-template--HasExpandAll}}
```
Comment (mattnolting) [inline src/patternfly/demos/Toolbar/toolbar-template.hbs:29]: I think you'll want to move this outside of the `toggle-group` for two reasons. 1) I _think_ the expand all button should remain visible when `toggle-group` hides its contents and 2) You would need to consider it in the expandable content section when `toggle-group` moves it there. 

@mmenestr @mcarrano does this sound right to you? Would you expect the expand all button to remain visible when filters are hidden?

Comment (mcoker) [reply src/patternfly/demos/Toolbar/toolbar-template.hbs]: @mattnolting good call, updated!

Comment (mmenestr) [reply src/patternfly/demos/Toolbar/toolbar-template.hbs]: What do you mean when filters are hidden? Like if there's no toolbar?

**Review by mattnolting — APPROVED** [review]
LPTM 👍

**Review by mcarrano — APPROVED** [review]
Looks great. Thanks @mcoker !

**Review by nicolethoen — APPROVED** [review]
LGTM!

**Pull request merged**